persistence/src/main/java/org/cbioportal/genome_nexus/persistence/internal/EnsemblRepositoryImpl.java

```diff
@@ -109,4 +111,34 @@ public EnsemblTranscript findOneByHugoSymbolIgnoreCase(String hugoSymbol, String
         }
         return null;
     }
+
+    @Override
+    public HashMap<String, String> getHugoSymbolToEntrezGeneIdMap() {
```

Comment:
Every time this method is called the `HashMap` is constructed over and over again. Can we rename this method to `initHugoSymbolToEntrezGeneIdMap`, make it private, and invoke this method only in the constructor? Ideally we should only expose (make public) the method `findEntrezGeneIdByHugoSymbol(String hugoSymbol)`.